common3.json

```diff
@@ -7419,91 +7419,91 @@
     },
     {
       "id" : 61866,
-      "text" : "Counter:　Enemy side has pokemon with atk > sp.atk"
+      "text" : "Counter: Enemy side has pokemon with atk > sp.atk"
```

Comment:
This is a good example of what I would want the formatting to be: [thing]: [details], a lot of strings do not have the colon and I think whether or not the original strings had a colon implied, it's very clear looking at them that there should be a colon on all of them and it will help us on research since it's more clear

Reply:
I added the colon and space on all translated strings, leaving those with untranslated move names or those having trailing spaces in the original